tests/test_opera_rtc_s1.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+
+import hyp3_sdk
+import pytest
+import requests
+from osgeo import gdal
+
+from hyp3_testing import util
+from hyp3_testing.helpers import archive_tifs, job_tifs
+from hyp3_testing.opera_compare import compare_rtc_hdf5_files, compare_rtc_s1_products
+
+
+gdal.UseExceptions()
+CMR_URL = 'https://cmr.earthdata.nasa.gov/search/granules.umm_json'
+pytestmark = pytest.mark.golden
+
+
+@pytest.mark.nameskip
+def test_golden_submission(comparison_environments):
+    comparison_environments = [comparison_environments[1]]
+    job_name = util.generate_job_name()
+    print(f'Job name: {job_name}')
+
+    testing_parameters = util.render_template('opera_s1_rtc_golden.json.j2', name=job_name)
+    submission_payload = [{k: item[k] for k in ['name', 'job_parameters', 'job_type']} for item in testing_parameters]
+
+    for dir_, api in comparison_environments:
+        dir_.mkdir(parents=True, exist_ok=True)
+
+        hyp3 = hyp3_sdk.HyP3(api, os.environ.get('EARTHDATA_LOGIN_USER'), os.environ.get('EARTHDATA_LOGIN_PASSWORD'))
+        jobs = hyp3.submit_prepared_jobs(submission_payload)
+        request_time = jobs.jobs[0].request_time.isoformat(timespec='seconds')
+        print(f'{dir_.name} request time: {request_time}')
+
+        submission_details = {'name': job_name, 'request_time': request_time}
+        submission_report = dir_ / f'{dir_.name}_submission.json'
+        submission_report.write_text(json.dumps(submission_details))
+
+
+@pytest.mark.timeout(3600)  # 60 minutes as RTC jobs timeout after 1 hour
+@pytest.mark.dependency()
+def test_golden_wait(comparison_environments, job_name, user_id):
+    comparison_environments = [comparison_environments[1]]
+    for dir_, api in comparison_environments:
+        if job_name is None:
+            submission_report = dir_ / f'{dir_.name}_submission.json'
+            submission_details = json.loads(submission_report.read_text())
+            job_name = submission_details['name']
+
+        hyp3 = hyp3_sdk.HyP3(api, os.environ.get('EARTHDATA_LOGIN_USER'), os.environ.get('EARTHDATA_LOGIN_PASSWORD'))
+        jobs = hyp3.find_jobs(name=job_name, user_id=user_id)
+
+        assert len(jobs) > 0  # will throw if job_name not associated with user_id
+
+        _ = hyp3.watch(jobs)
+
+
+@pytest.mark.dependency(depends=['test_golden_wait'])
+def test_golden_job_succeeds(develop_jobs_info):
+    develop_succeeds = sum([value['develop']['succeeded'] for value in develop_jobs_info.values()])
+    assert develop_succeeds != 0
```

Comment:
I know this function is adapted from our existing [`test_golden_job_succeeds`](https://github.com/ASFHyP3/hyp3-testing/blob/develop/tests/test_rtc.py#L56) in `test_rtc.py`, but what's the reason for asserting `develop_succeeds != 0` instead of asserting something like `all([value['develop']['succeeded'] for value in develop_jobs_info.values()])`. Don't we want to confirm that all jobs succeeded?